packages/fxa-content-server/tests/functional/support.js

```diff
@@ -36,7 +35,8 @@ registerSuite('support form without valid session', {
 
 registerSuite('support form without active subscriptions', {
   tests: {
-    'go to support form without active subscriptions, redirects to subscription management': function() {
+    'go to support form, redirects to subscription management': function() {
+      let email = TestHelpers.createEmail();
```

Comment:
hm does this have to be a `let` here? and below?

Reply:
@chenba yes see CI log:

> /home/circleci/project/packages/fxa-content-server/tests/functional/support.js
  39:11  error  'email' is never reassigned. Use 'const' instead  prefer-const
  54:11  error  'email' is never reassigned. Use 'const' instead  prefer-const

Reply:
Yeah, just plain ol' mistake. :(